sources/model/entity/Event.ts

```diff
@@ -9,29 +9,29 @@ export class Event {
     @PrimaryGeneratedColumn()
     id: number;
 
-    @Column({length: 256})
+    @Column({ length: 256 })
     name: string;
 
-    @Column()
+    @Column({ nullable: true })
     description: string;
 
-    @Column()
+    @Column({ nullable: true })
     location: string;
```

Comment:
So `location` is not mandatory anymore ?

If so please update the check in event creation and edition